src/web/modules/services/detail.njk

```diff
@@ -103,6 +103,11 @@
     </table>
   </div>
 
+  <div>
```

Comment:
This may change to a button but if not suggest the following:
```suggestion
  <div class="govuk-!-padding-bottom-8 govuk-body">
```
* `govuk-body` will make the link the same size as the other links on the page
* padding is fine to go on the parent div

Reply:
Do you mean like this?
```
  <div class="govuk-!-padding-bottom-8">
    <a href="mailto:{{ adminEmails }}?cc=govuk-pay-support@digital.cabinet-office.gov.uk" class="govuk-body">Email all admin users</a>
  </div>
```
(This works by the way)